Editor/CoreAPI/GameLiftFleetManager.cs

```diff
@@ -45,74 +45,45 @@ public async Task<CreateAnywhereFleetResponse> CreateFleet(string fleetName)
                 return Response.Fail(new CreateAnywhereFleetResponse { ErrorCode = ErrorCode.InvalidFleetName });
             }
 
-            var createLocationSuccess = await CreateCustomLocationIfNotExists(FleetLocation);
-            if (!createLocationSuccess)
-            {
-                return Response.Fail(new CreateAnywhereFleetResponse { ErrorCode = ErrorCode.CustomLocationCreationFailed });
-            }
-
-            var fleetId = await CreateFleet(ComputeType.ANYWHERE, FleetLocation, fleetName);
-            if (string.IsNullOrWhiteSpace(fleetId))
-            {
-                return Response.Fail(new CreateAnywhereFleetResponse { ErrorCode = ErrorCode.CreateFleetFailed });
-            }
-
-            return Response.Ok(new CreateAnywhereFleetResponse()
-            {
-                FleetId = fleetId,
-                FleetName = fleetName
-            });
-        }
-
-        private async Task<bool> CreateCustomLocationIfNotExists(string fleetLocation)
```

Comment:
I'm not a fan of making this one large method with two try-catches. What if the two helper methods return a Response object. CreateCustomLocationIfNotExists's response can include just the ErrorMessage while CreateFleet can return the same response object as the overall method (no need to create a new one, just return the result of `await CreateFleet(...)`).

Also consider updating CreateAnywhereFleetResponse to CreateFleetResponse.